### DISCUSSION TITLE
2.0.9修改配置重启闪退bug

### DISCUSSION THREAD

**Discussion by ljm12914** [libsgh/PanIndex#218] [Q&A]
复现Bug方式：第一次启动程序后，使用`http://localhost:5238`进入管理界面，将“基础配置——绑定Host”修改为域名，保存，然后重启程序。程序将闪退。没有显示任何错误提示。

其他信息：Windows Server 2008，目录内无任何其他文件，无自定义UI。将`data.db`与正常的文件进行对比，暂时没有发现明显差异。

PS:重启程序闪退在修改完配置后其实很常见，不仅有这一种方式，不仅有这一个版本会这样。

**Comment by libsgh** [comment]
HOST配置的域名是什么样的

**Reply by ljm12914** [reply]
subdomain.domain.com
